### DISCUSSION TITLE
How to play rtsp video using ft.video()

### DISCUSSION THREAD

**Discussion by hi-one-gg** [flet-dev/flet#3145] [Q&A]
I am very pleased and grateful for the addition of video control to flet.
I wanted to play an rtsp video using video control, but after a brief loading UI, the video does not play.

Please tell me how to play an rtsp video.

### ENV.
Windows11
flet 0.21.2 and 0.22.0

**Comment by KunMengcode** [comment]
@blackCmd https://flet.dev/docs/controls/video
